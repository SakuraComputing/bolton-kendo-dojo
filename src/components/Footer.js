import React from 'react'

const Footer = () => {
  return (
    <div className="footer">
      <div className="bka-details">
        <a className="bka-link" href="http://www.britishkendoassociation.com" target="_blank" rel="noopener noreferrer">Member of the British Kendo Association<img src="/img/bka_logo3.png" alt="BKA Logo" className="bka-logo"/></a>
      </div>
      <div className="copyright">
        Copyright &copy; {new Date().getFullYear()} Bolton Kendo Club
      </div>
    </div>
  )
}
export default Footer;
