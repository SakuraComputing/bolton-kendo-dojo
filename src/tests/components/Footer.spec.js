import { shallow } from 'enzyme';
import React from 'react';
import Footer from '../../components/Footer.js'

describe('Footer component', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<Footer />);
    });

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});