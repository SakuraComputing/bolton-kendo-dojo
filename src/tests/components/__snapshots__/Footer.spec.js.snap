// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="bka-details"
        >
          <a
            className="bka-link"
            href="http://www.britishkendoassociation.com"
            rel="noopener noreferrer"
            target="_blank"
          >
            Member of the British Kendo Association
            <img
              alt="BKA Logo"
              className="bka-logo"
              src="/img/bka_logo3.png"
            />
          </a>
        </div>,
        <div
          className="copyright"
        >
          Copyright © 
          2019
           Bolton Kendo Club
        </div>,
      ],
      "className": "footer",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <a
            className="bka-link"
            href="http://www.britishkendoassociation.com"
            rel="noopener noreferrer"
            target="_blank"
          >
            Member of the British Kendo Association
            <img
              alt="BKA Logo"
              className="bka-logo"
              src="/img/bka_logo3.png"
            />
          </a>,
          "className": "bka-details",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Member of the British Kendo Association",
              <img
                alt="BKA Logo"
                className="bka-logo"
                src="/img/bka_logo3.png"
              />,
            ],
            "className": "bka-link",
            "href": "http://www.britishkendoassociation.com",
            "rel": "noopener noreferrer",
            "target": "_blank",
          },
          "ref": null,
          "rendered": Array [
            "Member of the British Kendo Association",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "BKA Logo",
                "className": "bka-logo",
                "src": "/img/bka_logo3.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
          ],
          "type": "a",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Copyright © ",
            2019,
            " Bolton Kendo Club",
          ],
          "className": "copyright",
        },
        "ref": null,
        "rendered": Array [
          "Copyright © ",
          2019,
          " Bolton Kendo Club",
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="bka-details"
          >
            <a
              className="bka-link"
              href="http://www.britishkendoassociation.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              Member of the British Kendo Association
              <img
                alt="BKA Logo"
                className="bka-logo"
                src="/img/bka_logo3.png"
              />
            </a>
          </div>,
          <div
            className="copyright"
          >
            Copyright © 
            2019
             Bolton Kendo Club
          </div>,
        ],
        "className": "footer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <a
              className="bka-link"
              href="http://www.britishkendoassociation.com"
              rel="noopener noreferrer"
              target="_blank"
            >
              Member of the British Kendo Association
              <img
                alt="BKA Logo"
                className="bka-logo"
                src="/img/bka_logo3.png"
              />
            </a>,
            "className": "bka-details",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "Member of the British Kendo Association",
                <img
                  alt="BKA Logo"
                  className="bka-logo"
                  src="/img/bka_logo3.png"
                />,
              ],
              "className": "bka-link",
              "href": "http://www.britishkendoassociation.com",
              "rel": "noopener noreferrer",
              "target": "_blank",
            },
            "ref": null,
            "rendered": Array [
              "Member of the British Kendo Association",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "BKA Logo",
                  "className": "bka-logo",
                  "src": "/img/bka_logo3.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
            ],
            "type": "a",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Copyright © ",
              2019,
              " Bolton Kendo Club",
            ],
            "className": "copyright",
          },
          "ref": null,
          "rendered": Array [
            "Copyright © ",
            2019,
            " Bolton Kendo Club",
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
